decred_wallet/view_controller/WalletSetupViewController.swift

```diff
@@ -15,6 +15,7 @@ class WalletSetupViewController : UIViewController {
     @IBOutlet weak var restoreWallet: UILabel!
     @IBOutlet weak var createWallet: UILabel!
     @IBOutlet weak var build: UILabel!
+    @IBOutlet weak var walletText: UILabel!
```

Comment:
This outlet is not used, why do we have it?

Reply:
It was initially added for font scaling in previous commit, but removed now